src/backend/utils/html/index.js

```diff
@@ -14,14 +15,16 @@ const { JSDOM } = jsdom;
  * modifies <img> and <iframe> elements to be lazy loaded. Returns
  * the updated HTML.
  */
-module.exports = function process(html) {
+module.exports = function process(html, type) {
   // Expect a String, return anything else untouched.
   if (typeof html !== 'string') {
     return html;
   }
 
+  const linkifiedHtml = type === 'video' ? linkifyHtml(html) : html;
```

Comment:
I don't think this is the right place to do this, as we expect HTML at this point.